charts/kubernikus-system/templates/prometheus-alerts.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: common-prometheus-rules
+  labels:
+    app: fusion
+    type: rule
```

Comment:
Is there another type except `rules`? I suppose, the `app` label is being used for selecting on the ConfigMaps that contain Prometheus-Fusion config. Isn't this double bubble with the annotation below?

Reply:
The `labels` of an alert/rule configmap would describe the app they belong to. So in Kubernetes it would be like
```
app: kubernikus
type: api
```
The above for the common rules is a bad example and should be changed to `app: Kubernetes`